searx/network/__init__.py

```diff
@@ -83,11 +84,27 @@ def request(method, url, **kwargs):
     network = get_context_network()
 
     # do request
-    future = asyncio.run_coroutine_threadsafe(network.request(method, url, **kwargs), get_loop())
-    try:
-        response = future.result(timeout)
-    except concurrent.futures.TimeoutError as e:
-        raise httpx.TimeoutException('Timeout', request=None) from e
+    if isinstance(url, str):
+        future = asyncio.run_coroutine_threadsafe(network.request(method, url, **kwargs), get_loop())
+        try:
+            response = future.result(timeout)
+        except concurrent.futures.TimeoutError as e:
+            raise httpx.TimeoutException('Timeout', request=None) from e
+    elif isinstance(url, list):
+        future_list = [asyncio.run_coroutine_threadsafe(network.request(method, u, **kwargs), get_loop()) for u in url]
+        responses = []
```

Comment:
> For what it's worth, I've deployed this PR on my instance, and the bing engine works fine since then (it is enabled in all my browsers).

OK, but I think this part of the `network.request` where the same `**kwargs` are send to a list of URLs is not a normal use case .. as you said: "quick & dirty" :) .. I think we should not merge this patch of `network.request`.